File: src/test/java/br/com/model/classes/NovoProdutoTest.java
package br.com.model.classes;

import org.junit.jupiter.api.Test;

import br.com.newObjects.model.classes.NovoProduto;

public class NovoProdutoTest {
    @Test
    void testNovoProduto() {
        System.out.println(NovoProduto.novoProduto().toString());
    }

    @Test
    void testNovoProduto2() {
        System.out.println(NovoProduto.novoProduto2().toString());
    }
}

File: src/test/java/br/com/model/NovoProdutoTest.kt
package br.com.model

import br.com.newObjects.model.NovoProduto
import org.junit.jupiter.api.Test

class NovoProdutoTest {
    @Test
    fun testNovoProduto() {
        println(NovoProduto.novoProduto().toString())
    }

    @Test
    fun testNovoProduto2() {
        println(NovoProduto.novoProduto2().toString())
    }
}
